contrib/mungegithub/blunderbuss.yml

```diff
@@ -0,0 +1,60 @@
+prefixMap:
+  test:
+    - ixdy
+
+  pkg/admission:
+    - lavalamp
+    - smarterclayton
+
+  pkg/api:
+    - bgrant0607
+
+  pkg/apiserver:
+    - lavalamp
+    - smarterclayton
+
+  pkg/authentication:
+    - liggitt
+    - erictune
+
+  pkg/client:
+    - lavalamp
```

Comment:
If you want smaller pools for each that's fine - I included folks who would
generally be suitable for triage or a first pass.

On Aug 8, 2015, at 1:40 AM, David Oppenheimer <notifications@github.com>
wrote:

In contrib/mungegithub/blunderbuss.yml
<https://github.com/GoogleCloudPlatform/kubernetes/pull/12365#discussion_r36575398>
:

> +    - lavalamp
> +    - smarterclayton
> +
> +  pkg/api:
> +    - bgrant0607
> +
> +  pkg/apiserver:
> +    - lavalamp
> +    - smarterclayton
> +
> +  pkg/authentication:
> +    - liggitt
> +    - erictune
> +
> +  pkg/client:
> +    - lavalamp

@smarterclayton <https://github.com/smarterclayton> just to be clear, this
isn't a list of who will be Cc'd on PRs in each directory -- it's the pool
of people the bot will randomly pick from to assign unassigned PRs that
touch each directory

—
Reply to this email directly or view it on GitHub
<https://github.com/GoogleCloudPlatform/kubernetes/pull/12365/files#r36575398>
.